db-scripts/src/main/resources/migration.sql

```diff
@@ -904,7 +904,8 @@ CREATE TABLE `alteration_driver_annotation` (
   `DRIVER_TIERS_FILTER_ANNOTATION` VARCHAR(80),
   PRIMARY KEY (`ALTERATION_EVENT_ID`, `GENETIC_PROFILE_ID`, `SAMPLE_ID`),
   FOREIGN KEY (`GENETIC_PROFILE_ID`) REFERENCES `genetic_profile` (`GENETIC_PROFILE_ID`) ON DELETE CASCADE,
-  FOREIGN KEY (`SAMPLE_ID`) REFERENCES `sample` (`INTERNAL_ID`) ON DELETE CASCADE
+  FOREIGN KEY (`SAMPLE_ID`) REFERENCES `sample` (`INTERNAL_ID`) ON DELETE CASCADE,
+  INDEX (`DRIVER_FILTER`)
 ) COMMENT='Alteration driver annotation';
```

Comment:
this won't work as migration if your already running 2.12.7 - maybe best to create a 2.12.8 migration and add indices to the existing table

Reply:
Ah yes, moved this to a new version section.